server/src/main/java/org/elasticsearch/cluster/coordination/NodeRemovalClusterStateTaskExecutor.java

```diff
@@ -59,25 +59,29 @@ public NodeRemovalClusterStateTaskExecutor(AllocationService allocationService)
     public ClusterTasksResult<Task> execute(final ClusterState currentState, final List<Task> tasks) throws Exception {
         final DiscoveryNodes.Builder remainingNodesBuilder = DiscoveryNodes.builder(currentState.nodes());
         boolean removed = false;
+        final ClusterTasksResult.Builder<Task> resultBuilder = ClusterTasksResult.builder();
```

Comment:
Nit: I would simplify to:

```suggestion
        var resultBuilder = ClusterTasksResult.<Task>builder();
```